BMS_CurrentRangeFrequency.test.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import random
+import numpy as np
+from BMS_CurrentRangeFrequency import CurrentRangeFrequncy
+
+min_current_reading = 0
+max_current_reading = 40
+sample_data_size = 20
+
+data = [random.randint(min_current_reading, max_current_reading) for x in range(sample_data_size)]
+crf = CurrentRangeFrequncy(data)
+random_index = random.randint(0, len(data))
+
+class CurrentRangeFrequencyTest(unittest.TestCase):
+    def test_CurrentRangeFrquency_output_desired_results(self):

```

Comment:
The core of the functionality is expressed in a testcase without asserts. The consequences of this are:
. The output must be seen manually - possibility of mistakes, and the tester must be aware of the implementation to judge correctness
. The output format isn't in the test. It will need to be documented somewhere else, which can go out of sync with the code, over time.

Write a test to check the output format.